### DISCUSSION TITLE
Store and Record audio in real time.

### DISCUSSION THREAD

**Discussion by ponikar** [dooboolab/react-native-audio-recorder-player#482] [Q&A]
Is there any way to store and record audio in real-time? (without stopping the recorder). 

I want to keep track of audio by storing it temporarily in a file without stopping the recorder. 

I am trying to extract the base64 string and strong into an audio file but it's not working the audio is getting corrupted.

**Comment by jforaker** [comment]
Same! Let me know if you find a workaround. My use case is to validate the file size in real time / or once per tick from `addRecordBackListener` 
Strange `addRecordBackListener` doesn't return the uri and only `stopRecorder` does.

**Comment by hyochan** [comment]
@jforaker What would be the pros if we add back the `uri` in `addRecordBackListener`?
It will receive the same `uri` parameter.

**Comment by ponikar** [comment]
My use case was something like this 

I am recording the audio now somehow the app got destroyed, and I want that recording to be merged with the new recording but the file which was basically stored in the cache gets corrupted.
